lib/org/openpsa/invoices/billing/data.php

```diff
@@ -33,6 +33,57 @@ public function _on_creating()
     }
 
     /**
+     * Get the billing data for the customer in object or create a default data set of it.
+     *
+     * @param org_openpsa_invoices_interfaces_customer $object
+     * @return org_openpsa_invoices_billing_data_dba|boolean
```

Comment:
Actually, the function currently cannot return `boolean`, so the typehint should be adjusted